src/api/core/componentStorage.spec.ts

```diff
@@ -1,10 +1,11 @@
 // tslint:disable:no-string-literal
+// @ts-ignore
```

Comment:
can you test adding `@types/faker` package to check if ts compiler would still complain?